src/data/versions/dev6.0.41r-1.json

```diff
@@ -125,7 +125,8 @@
             ],
             "Shuffle Child Trade": [
                 "Off",
-                "On"
+                "Basic",
```

Comment:
If this is going to change from an on/off switch, it should be made a multi-select for every possible child trade item to align with the upstream randomizer setting.

Reply:
i agree, and same goes for shuffle adult trade. this was a really lazy implementation that assumes only you either have just the first item or all of them. not sure how to make it a multiselect but i'll look into it